CannibalisticZombies/Assets/Scripts/HealthUI.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class HealthUI : MonoBehaviour
+{
+    // HealthUI displays current health.
+    // Calling SetHealth() with new Damage amount update healthBar to that amount.
+    public Image healthBar;
+    public float currentHealth;//unnecessary in implementation
+    public float animationSpeed; //Rate at which damage bar fills/empties per frame
+    public Color damageColor=Color.red;//Color of bar when take damage
```

Comment:
For code readability, make sure to put spaces between `=` and the neighboring code.  And to also put a space between the comment and the code.